engine/battle/core.asm

```diff
@@ -2557,11 +2557,12 @@ MoveSelectionMenu:
 	ld a, [wLinkState]
 	cp LINK_STATE_BATTLING
 	jr z, .matchedkeyspicked
+	; debug, TestBattle-related, unclear purpose 
```

Comment:
Just wait til you have enough info, and rename the `wFlags_D733` label accordingly. This is just a fancy TODO comment.

Reply:
> Just wait til you have enough info

I hadn't noticed any visible difference between regular and test battles at the time.